src/tetris.h

```diff
@@ -10,11 +10,11 @@ class Tetris {
     void run();
 
   private:
-    bool EventTriggered(double interval);
+    bool EventTriggered(float interval);
```

Comment:
prefer to use float instead of double when possible because float is usually faster. Not an issue here but should be noted anyway: Microcontrollers used in Embedded Devices are usually equipped with a HARDWARE Floating Point unit that often just supplies float (single precision float numbers). A double would **not** employ these but use expensive Software routines! So its a good practice to use `float` instead of `double` when appropriate!